File: app/src/main/java/at/priv/graf/base/SpinnerUtil.java
package at.priv.graf.base;

import at.priv.graf.zazentimer.bo.Session;
import java.util.ArrayList;

public class SpinnerUtil {
    public static int getPositionById(ArrayList<Session> arrayList, int i) {
        for (int i2 = 0; i2 < arrayList.size(); i2++) {
            if (arrayList.get(i2).id == i) {
                return i2;
            }
        }
        return -1;
    }
}

File: app/src/main/kotlin/at/priv/graf/zazentimer/base/SpinnerUtil.kt
package at.priv.graf.zazentimer.base

import at.priv.graf.zazentimer.bo.Session

object SpinnerUtil {
    @JvmStatic
    fun getPositionById(
        list: ArrayList<Session>,
        id: Int,
    ): Int {
        for (i in list.indices) {
            if (list[i].id == id) {
                return i
            }
        }
        return -1
    }
}
